core/src/main/scala/kafka/zk/KafkaZkClient.scala

```diff
@@ -82,6 +82,12 @@ class KafkaZkClient private (zooKeeperClient: ZooKeeperClient, isSecure: Boolean
     info(s"Registered broker ${brokerInfo.broker.id} at path $path with addresses: ${brokerInfo.broker.endPoints}")
   }
 
+  def registerControllerInZk(brokerId: Int, timestamp: Long): Unit = {
```

Comment:
An alternative is to use the time object of the `KafkaZkClient` to get the timestamp.
At the moment KafkaController uses the same time object and does not use the timestamp for anything else but to create the zk node, so it looks like a safe change.

Reply:
Can we please document the method?